### DISCUSSION TITLE
Having trouble connecting to github.com

### DISCUSSION THREAD

**Discussion by prashanthsn** [community/community#41347] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

ping -n 10 github.com

Pinging github.com [20.207.73.82] with 32 bytes of data:
Request timed out.
Request timed out.
Request timed out.
Request timed out.
Request timed out.
Request timed out.
Request timed out.
Request timed out.
Request timed out.
Request timed out.

Ping statistics for 20.207.73.82:
    Packets: Sent = 10, Received = 0, Lost = 10 (100% loss),

Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36

                
f10ab0a17781c913edb35f7a3999c821

                
10A2:0837:115A:81524:6394BE24

                
                

                  
traceroute to 49.37.249.243 (49.37.249.243), 30 hops max, 60 byte packets
 1  *
 2  *
 3  *
 4  *
 5  *
 6  *
 7  *
 8  *
 9  *
10  *
11  *
12  *
13  *
14  *
15  *
16  *
17  *
18  *
19  *
20  *
21  *
22  *
23  *
24  *
25  *
26  *
27  *
28  *
29  *
30  *

                  
PING 49.37.249.243 (49.37.249.243) 56(84) bytes of data.

--- 49.37.249.243 ping statistics ---
3 packets transmitted, 0 received, 100% packet loss, time 2039ms

                  
Start: 2022-12-10T09:13:24-0800
HOST: e5973ff74adda0cdd444c88380120546               Loss%   Snt   Last   Avg  Best  Wrst StDev

                  
Host 243.249.37.49.in-addr.arpa. not found: 3(NXDOMAIN)

**Comment by jfullstackdev** [comment]
it's intentional I think so, same results on my end

![image](https://user-images.githubusercontent.com/47092464/206867482-ba1509c9-7a58-44a0-b6d7-d4f492a790cf.png)

but I'm connected to GitHub right now, you also, for sure, you were able to post this

**Reply by prashanthsn** [reply]
My issue is that the github dependent 2 tools are not able to connect to the server without a VPN. Wabbajack and Universal Android Debloater. New to github and not sure where to check for these issues.
